x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/search/TokenPruningConfig.java

```diff
@@ -21,7 +22,7 @@
 import java.util.Objects;
 import java.util.Set;
 
-import static org.elasticsearch.xpack.ml.queries.TextExpansionQueryBuilder.PRUNING_CONFIG;
+import static org.elasticsearch.xpack.core.ml.search.WeightedTokensQueryBuilder.PRUNING_CONFIG;
```

Comment:
Nit, and probably out of scope in this PR, but using `PRUNING_CONFIG` to generate exception messages in this class assumes that all upstream usage is via a field named "pruning_config". Which is true right now, but it may not stay that way...

Reply:
Let's save future use refactoring for future PRs.